src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -28,10 +29,68 @@ export const calculateTableEndYPosition = (table: TableVm): number => {
   return table.y + tableHeight;
 };
 
+export const doesTableOverlap = (
```

Comment:
Same thing here this code is not related to the case, and likely you are stuck on older version of vnext?